week-5/Homework/mandatory/2-exercises/exercises.js

```diff
@@ -1,66 +1,58 @@
-/**
- *
- * For each of the names in the array passed into this function
- *
- * - Add a <h1> tag to the website containing the name of the person
- * - Add a <h2> tag to the website containing the job of the person
- *
- * All of your HTML should go inside the Div tag with the id "content".
- *
- * <div id="content">
- *      <h1>{Name Here}</h1>
- *      <h2>{Job Here}</h2>
- *      .....
- * </div>
- */
+let myHeadOne = document.createElement("h1");
+let myHeadtwo = document.createElement("h2");
 function exerciseOne(arrayOfPeople) {
   let content = document.querySelector("#content");
+  content.appendChild(myHeadOne);
+  content.appendChild(myHeadtwo);
+  let names =[];
+   arrayOfPeople.filter(x => names.push(x.name));
+  let jobs = [];
+  arrayOfPeople.filter(x =>jobs.push(x.job));
+  for(let i = 0 ; i<names.length ; i++){
+    myHeadOne.appendChild(document.createTextNode(" "+names[i]+" /"));
+    myHeadtwo.appendChild(document.createTextNode(" " +jobs[i]+ " /"));
+  }
 }
 
-/**
- *
- * Create a list of shopping items. You should use an unordered list.
- *
- * All of your HTML should go inside the Div tag with the id "content".
- *
- */
-function exerciseTwo(shopping) {
-  //Write your code in here
-}
-
-/**
-    I'd like to display my three favorite books inside a nice webpage!
-
-    const books = [
-        {
-            title: "The Design of Everyday Things",
-            author: "Don Norman",
-            alreadyRead: false
-        },
-        {
-            title: "The Most Human Human",
-            author: "Brian Christian",
-            alreadyRead: true
-        },
-        {
-            title: "The Pragmatic Programmer",
-            author: "Andrew Hunt",
-            alreadyRead: true
-        }
-    ];
 
-    Iterate through the array of books.
-    - For each book, create a <p> element with the book title and author and append it to the page.
-    - Use a <ul> and <li> to display the books.
-    - Add an <img> to each book that links to a URL of the book cover.
-    - Change the style of the book depending on whether you have read it (green) or not (red).
 
-    The end result should look something like this: https://hyf-js2-week1-makeme-ex1-demo.herokuapp.com/
-**/
-function exerciseThree(books) {
-  //Write your code in here
+function exerciseTwo(shopping) {
+  let content = document.querySelector("#content");
+  let unorderedList = document.createElement("ul");
+  content.appendChild(unorderedList);
+  for(let i = 0;i< shopping.length;i++){
```

Comment:
A few small auto-alignment issues here. Have you tried using prettier?

https://github.com/prettier/prettier-vscode

Reply:
I have prettier and I used it but I don't know why this little red sign doesn't go 
![Screenshot 2020-04-15 at 16 57 52](https://user-images.githubusercontent.com/59767576/79360196-791d9000-7f3b-11ea-960f-94c6870673dd.png)